script.js

```diff
@@ -0,0 +1,31 @@
+//const sayHello = () => alert('Hello world!');
+//sayHello();
+
+//zad 1
+const first = 'Hello ';
+const second = 'World';
+console.log(`${first} ${second}`);
+
+//zad 2
+const multiply = (a = '1', b = '1') => console.log(`${a * b}`)
```

Comment:
Mnożymy liczby a nie łańcuchy znaków, czyli bez apostrofów, fakt, że to akurat działa to wynik rzutowania JS który próbuje sobie poradzić z wyrażeniem, ale takie użycie jest błędem i trzeba poprawnie określić wartość

Reply:
poprawione ;)

Reply:
Nie bardzo, ciągle wartości domyślne masz podane jako stringi

Reply:
poprawione po raz drugi